web/src/assets/styles/patternfly-overrides.scss

```diff
@@ -165,3 +165,8 @@ table td > .pf-c-empty-state {
   outline: none;
   box-shadow: 0 0 0 1px var(--focus-color);
 }
+
+// Center icon in the visibility button of password input form fields
+button#password_visibility span.pf-c-button__icon {
```

Comment:
This selector won't work anymore, since the `id` is now changeable and not static `#password_visibility`. I still link, that a `class` on the component and in this selector is the easiest way to fix this.

Reply:
You're right, the selector as it is won't work. Good catch 😉 Probably,  a class will be needed too,as you mention. A `password-toggler` or better suggestion if you have something in mind.

But keeping the `id` attribute in the button. I'm not a big fan of adding `id`s if they are not strictly needed, but somehow we've commited internally on having form elements reachable by id.

Thank you for taking care and reviewing the PR 💯